cl/users/tests.py

```diff
@@ -2142,3 +2143,37 @@ def test_avoid_add_bcc_to_emails(self) -> None:
         self.assertEqual(
             stored_email[1].bcc, ["bcc@example.com", "bcc@example.com"]
         )
+
+
+class DeleteOldMessagesCommandTest(TestCase):
+    """Test cl_delete_old_messages command"""
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.msg1 = EmailSentFactory()
+        cls.msg2 = EmailSentFactory()
+        cls.msg3 = EmailSentFactory()
+
+    def test_delete_old_messages(self):
+        """This test checks if cl_delete_old_messages command works properly
+        it should delete only messages older than the specified number of days.
+        """
+
+        # Update date_created to simulate two messages are older than 15 days
+        self.msg1.date_created = now() - timedelta(days=16)
+        self.msg1.save()
+        self.msg2.date_created = now() - timedelta(days=16)
+        self.msg2.save()
+
```

Comment:
Why not incude the date_created in your set up?

Reply:
sure, I moved it to setUpTestData